src/cobalt.rs

```diff
@@ -120,6 +124,57 @@ pub fn build(config: &Config) -> Result<()> {
         }
     }
 
+    // load data files
+    let mut data_map : HashMap<String, Value> = HashMap::new();
+    let data_root = source.join("_data");
+    let data_files_builder = FilesBuilder::new(data_root.as_path())?;
+    let data_files = data_files_builder.build()?;
+
+    for df in data_files.files() {
+
+        if let Some(ext) = df.extension() {
+            if let Some(file_stem) = df.file_stem() {
+                let file_name = String::from(file_stem.to_str().unwrap());
+                let full_path = data_root.join(df.clone());
+                let data : Value;
+
+                if ext == OsStr::new("yml") || ext == OsStr::new("yaml")  {
```

Comment:
> add support for CSV (I have a in-progress-stash about that)

If there is nothing contentious in the design (like configuring the type of csv it reads) and trivial, let's do it.

Reply:
After playing a bit more with it, the CSV reader is quite a bit more annoying to use (no direct serde-support like all the others here have) and makes the code base a lot more complex at the moment..

Considering that I haven't ever seen this being used in production, I'm tended to skip it for this PR and add open a ticket about it afterwards...

Reply:
Ok